src/BatteryStateOfChargeService.cpp

```diff
@@ -1,13 +1,21 @@
 #include "BatteryStateOfChargeService.h"
+#include <QDebug>
+#include <QFile>
+#include <QString>
+#include <QTextStream>
```

Comment:
So these 4 new includes are not being used in this file so they don't need to be included.